boards/DCM/Test/Src/Test_StateMachine.cpp

```diff
@@ -427,54 +438,79 @@ TEST_F(
     App_CanRx_DIM_REGEN_PADDLE_SetSignal_MAPPED_PADDLE_POSITION(
         can_rx_interface, 50.0f);
 
+    float expected_torque_request_value =
+        60.0f / 100.0f * MAX_SAFE_TORQUE_REQUEST_NM;
+    float expected_regen_request_value =
```

Comment:
Do you have any idea whether if the max regen torque request is capped at 21.0Nm as well?

Reply:
Not too sure about this